BUILD.bazel

```diff
@@ -287,6 +328,7 @@ cc_library(
     deps = [
         ":gcs",
         ":object_manager_fbs",
```

Comment:
It would be ok to move it into https://github.com/ray-project/ray/blob/master/src/ray/raylet/format/node_manager.fbs if you want to. I guess eventually everything will be converted to protobuf so we can also do it later.